### ISSUE TITLE
Hitting Windows key while client is focused results in both client and Linux server responding

### ISSUE THREAD

**Issue by ghost** [input-leap/input-leap#541]
> This issue has been migrated from old Barrier Github repository https://github.com/debauchee/barrier/issues/541
> 
> Issue created on: 2020-01-11 by @GitHubRub
> Issue last updated on: 2020-09-04

### Operating Systems ###

Server: Kubuntu 19.10

Client: Windows 7

### Barrier Version ###

2.3.2 Stable on both machines

### Steps to reproduce bug ###

1. Move mouse cursor to client monitor.
2. Press Windows key.
3. Client PC brings up Start menu, but...
4. Server PC also brings up application launcher.

**NOTE:** other commands that use the Windows key (such as Win + Tab for shifting through windows and Win + Up to resize current window) are correctly only acknowledged by the client, not the server. It is only pressing the Windows key by itself that incorrectly causes the Linux server to respond.

### Other info ###

* When did the problem start to occur?
As soon as I started using barrier.

* Is there a way to work around it?
Not that I've found.

* Does this bug prevent you from using Barrier entirely?
Not yet.

---

> Commented on: 2020-01-16 by @eka

Same happens to me with MacOS client and Windows 10 as server... if MacOS I hit Windows Key + L in Firefox (to focus the address bar) it will Lock the Windows 10 server, in Windows 10 Windows Key + L is Lock machine.

v2.3.2

Also in MacOS my Command and Option keys switched places.

---

> Commented on: 2020-09-04 by @kepi

`Win  + L` is driving me crazy as I'm using it on client to move to right workspace and it always locks the server (windows) and move me there. For me, it unfortunately makes barrier unusable at the moment :(

**Comment by benmordecai** [comment]
Has there been anything done or any progress made on this? This is the main remaining problem I have with barrier and I have been considering a switch to input leap if issues like this get fixed in the future.

**Comment by shymega** [comment]
No, nothing yet. If there are no comments here, you can assume from that, that is has not been resolved.
